### DISCUSSION TITLE
[Question] How does Filebrowser scale?

### DISCUSSION THREAD

**Discussion by euh2** [filebrowser/filebrowser#1282] [Q&A] [answered]
I'm about to test this app and searched issues to find an answer. 

How is the behavior of Filebrowser when scaled? All instances would share `filebrowser.db` and a shared volume for userdata. Would that work?

In my case the instances would be loadbalanced trough kubernetes.

**Comment by o1egl** [comment, marked as answer]
The DB can't be shared among the instances. So at the moment, the service is not scalable. We don't plan to support massive scale. It is out of the scope of this project.
